client/src/Style.module.css

```diff
@@ -4,11 +4,12 @@ body {
     'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100vh;
```

Comment:
`vh` doesn't work in mobile browser as expected due to address bar. Use % instead.

Reply:
I have used percentage now. 
But I had to add an inline style in `client/public/index.html` to the div element with id root. I don't know why but adding them in `src/Style.module.css` wasn't reflecting the changes.